core/src/main/java/org/apache/druid/data/input/impl/JsonInputFormat.java

```diff
@@ -85,6 +106,8 @@ public JsonInputFormat(
       objectMapper.configure(feature, entry.getValue());
     }
     this.lineSplittable = lineSplittable;
+    this.assumeNewlineDelimited = assumeNewlineDelimited != null && assumeNewlineDelimited;
+    this.useJsonNodeReader = useJsonNodeReader != null && useJsonNodeReader;
```

Comment:
The description for this property above seems to indicate that it should be used for non-newline delimited JSON. Would `assumeNewLineDelimted` true interfere with this / should this be forced to false if `assumeNewLineDelimted` is true

Reply:
seems like maybe it would be an error to indicate `useJsonNodeReader` if `assumeNewLineDelimted` is set, though ignoring it seems ok too :shrug:

Reply:
okay, that makes sense, I'll have it be an error